pyvista/core/pointset.py

```diff
@@ -1600,10 +1601,10 @@ def linear_copy(self, deep=False):
 
         Converts the following cell types to their linear equivalents.
```

Comment:
I can count 10 `QUADRATIC_*` cell types and some more bi- and tri-quadratic ones. This method claims
> Return a copy of the unstructured grid containing only linear cells.

Do you think we need a feature request to track treating all the other (presumably newer) nonlinear cell types here eventually?

Reply:
See #3192